content/100-getting-started/index.mdx

```diff
@@ -1,108 +1,169 @@
 ---
-title: 'Get started'
-metaTitle: 'Get started'
-metaDescription: 'Get started'
+title: 'Get Started'
+metaTitle: 'Get started with Prisma'
+metaDescription: 'Build data-driven applications with ease using Prisma ORM, add connection pooling or global caching with Prisma Accelerate or subscribe to database changes in real-time with Prisma Pulse.'
+hideTitle: true
+tocDepth: 1
 ---
 
 <TopBlock>
 
-Welcome to the Prisma getting started section! 👋
+<BorderBox>
 
-Explore our range of products defined to make working with data easy:
+<BoxTitle>Get started</BoxTitle>
 
-[**Prisma ORM**](/orm/overview/introduction/what-is-prisma) is an [open source](https://github.com/prisma/prisma) next-generation Node.js and TypeScript ORM that unlocks a new level of developer experience when working with databases thanks to its intuitive data model, automated migrations, type-safety & auto-completion.
+Welcome 👋
 
-[**Prisma Accelerate**](/accelerate/what-is-accelerate) is a global database cache with scalable connection pooling.
+Explore our products that make it easy to build and scale data-driven applications:
 
-[**Prisma Pulse**](/pulse/what-is-pulse) allows you to build reactive, real-time applications in a type-safe manner.
+[**<Database />Prisma ORM**](/orm/overview/introduction/what-is-prisma) is a next-generation Node.js and TypeScript ORM that unlocks a new level of developer experience when working with databases thanks to its intuitive data model, automated migrations, type-safety & auto-completion.
```

Comment:
I wonder if/how we should change this link to be what-is-prisma-orm this probably involve changing the page title itself which also would mess up links. But its just a thought.